TinyConstraints/Classes/TinyConstraints.swift

```diff
@@ -339,7 +339,7 @@ public extension Constrainable {
     }
 }
 
-public extension TinyView {
+@MainActor public extension TinyView {
```

Comment:
Not needed since `TinyView` is already isolated to the `@MainActor`

```suggestion
public extension TinyView {
```